code/datums/elements/turf_transparency.dm

```diff
@@ -47,16 +61,44 @@
 		our_turf.underlays += plating_underlay
 	return TRUE
 
-/datum/element/turf_z_transparency/proc/on_multiz_turf_del(turf/our_turf, turf/T, dir)
+/datum/element/turf_z_transparency/proc/on_update_overlays(turf/our_turf, list/new_overlays_list)
 	SIGNAL_HANDLER
+
+	var/turf/below_turf = our_turf.below()
+	if(!below_turf)
+		return
+
+	var/datum/lighting_object/below_lighting = below_turf.lighting_object
+	var/mutable_appearance/below_turf_without_lighting = new(below_turf)
+	//below_turf_without_lighting.layer = our_turf.layer - 10
+	//below_turf_without_lighting.plane = our_turf.plane - 1
```

Comment:
was leaving this in intended for some reason?

Reply:
oops shit